src/components/Form/Form.js

```diff
@@ -211,9 +211,7 @@ export function Form({ children, config }) {
       // data-tid is helpful for writing tests.
       // sometimes it's passed in, but if it isn't,
       // we will automatically generate one
-      'data-tid':
-        fieldConfig.tid ||
-        [config.formName, config.formId, fieldName].join('-'),
+      'data-tid': fieldConfig.tid || [config.formId, fieldName].join('-'),
```

Comment:
FYI - not a huge deal, but removing `config.formName` broke some tests. Ultimately a good thing because it pushed me to use `fieldConfig.tid` instead, which seems more resilient.